packages/terra-dropdown-button/src/_variables.scss

```diff
@@ -0,0 +1,92 @@
+  active-caret-icon-background-image: var(--terra-dropdown-button-caret-active-background-image-neutral, inline-svg('<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" class="is-bidi"><path fill="#fff" d="M48 12L24 36 0 12h48z"/></svg>')),
+  disabled-caret-icon-background-image: var(--terra-dropdown-button-caret-disabled-background-image-neutral, inline-svg('<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" class="is-bidi"><path fill="#1c1f21" d="M48 12L24 36 0 12h48z"/></svg>'))
+);
+
+$terra-dropdown-button-emphasis-color-scheme: (
+  active-background-color: var(--terra-dropdown-button-active-background-color-emphasis, #004c76),
+  active-border-color: var(--terra-dropdown-button-active-border-color-emphasis, #0065a3),
+  active-color: var(--terra-dropdown-button-active-color-emphasis, #fff),
+  active-dropdown-font-weight: var(--terra-dropdown-button-dropdown-type-active-font-weight-emphasis, 400),
+  //active-split-font-weight: var(--terra-dropdown-button-split-type-active-font-weight-emphasis, 400),
+  //active-split-caret-box-shadow: var(--terra-dropdown-button-split-type-caret-active-box-shadow-emphasis, none),
+  background-color: var(--terra-dropdown-button-background-color-emphasis, #0079be),
+  border-color: var(--terra-dropdown-button-border-color-emphasis, #0065a3),
+  color: var(--terra-dropdown-button-color-emphasis, #fff),
+  dropdown-box-shadow: var(--terra-dropdown-button-dropdown-type-box-shadow-emphasis, none),
+  dropdown-font-weight: var(--terra-dropdown-button-dropdown-type-font-weight-emphasis, 400),
+  //split-box-shadow: var(--terra-dropdown-button-split-type-box-shadow-emphasis, none),
+  //split-font-weight: var(--terra-dropdown-button-split-type-font-weight-emphasis, 400),
+  //split-caret-box-shadow: var(--terra-dropdown-button-split-type-caret-box-shadow-emphasis, inset 1px 0 0 0 rgba(255, 255, 255, 0.2)),
+  disabled-color: var(--terra-dropdown-button-disabled-color-emphasis, #fff),
+  focus-background-color: var(--terra-dropdown-button-focus-background-color-emphasis, #0079be),
+  focus-border-color: var(--terra-dropdown-button-focus-border-color-emphasis, #0065a3),
+  focus-dropdown-box-shadow: var(--terra-dropdown-button-dropdown-type-focus-box-shadow-emphasis, 0 0 0 1px rgba(76, 178, 233, 0.7), 0 0 0 3px rgba(76, 178, 233, 0.35)),
+  //focus-split-box-shadow: var(--terra-dropdown-button-split-type-focus-box-shadow-emphasis, 0 0 0 1px rgba(76, 178, 233, 0.7), 0 0 0 3px rgba(76, 178, 233, 0.35)),
+  //focus-split-caret-box-shadow: var(--terra-dropdown-button-split-type-caret-focus-box-shadow-emphasis, 0 0 0 1px rgba(76, 178, 233, 0.7), 0 0 0 3px rgba(76, 178, 233, 0.35)),
+  hover-background-color: var(--terra-dropdown-button-hover-background-color-emphasis, #0065a3),
+  hover-color: var(--terra-dropdown-button-hover-color-emphasis, #fff),
+  //hover-split-caret-box-shadow: var(--terra-dropdown-button-split-type-caret-hover-box-shadow-emphasis, none),
```

Comment:
Let's remove this commented out code.

Reply:
Updated.

Reply:
these are the emphasis variant enhancement for split button that is ready to go for a quick-add after UX gets a chance to review the enhancement request. Have it stashed so fine to remove.